docs/adr/000x-modelling-history.md

```diff
@@ -0,0 +1,250 @@
+# X. Modelling history
+
+Date: 2018-12-17  
+Amended: 2019-01-29
+
+## Context
+
+The Content Publisher domain model is focused on storing current
+information. There is a documents table which stores the most recent content of
+a document and an images table which stores the most recent version of
+a document's images. History of these changes is stored using [Papertrail][],
+which is not [intended as a permanent store][papertrail-pr]. The presentation
+of a document's history is done via a TimelineEntry model which stores that a
+user did something, but not what they did. In places where richer information
+was required there have been models such as Removal or Withdrawal associated
+with a TimelineEntry.
+
+This has led to a number of pain points:
+
+- users cannot discard a draft of a document that has between previously
```

Comment:
I think you mean `been` here instead of `between` or `has previously been published` ?

Reply:
Oops! thanks 👍